### PR TITLE
using default instead of null

### PR DESCRIPTION
**Pull request by kylebuch8**
@castastrophe Let me know what you think.

I prefer seeing the word default in the dropdown as opposed to an empty field.

### CODE REVIEW

File: .storybook/utils.js [hunk 1]
```diff
@@ -194,7 +194,7 @@ export function autoPropKnobs(properties, bridge) {
 
         // If this is not a required field, add a null option
         if (!required) {
-          opts.null = "";
+          opts[""] = "default";
```
Comment (castastrophe) [inline .storybook/utils.js]: @kylebuch8 I'm fine if the printed value is default but the stored value should stay `null` for the rest of the logic in the dynamic build to work.

File: elements/pfe-card/src/pfe-card.story.js [hunk 2]
```diff
@@ -85,7 +85,6 @@ stories.add(PfeCard.tag, () => {
     const ctaPriorityValue = storybookBridge.select(
       "Priority",
       {
-        null: "",
```
Comment (castastrophe) [inline elements/pfe-card/src/pfe-card.story.js]: This is manually building the select field and not using the storybook utils I wrote so if you want it to include the default value, this needs to stay: `null: "default"`

File: .storybook/utils.js (hunk 1 above)
Comment (castastrophe) [reply .storybook/utils.js]: i.e., `opts.null = "default"`

**Comment by kylebuch8** [comment]
Fixed in https://github.com/patternfly/patternfly-elements/pull/277/commits/adf90c6aca8b31fd25a24741a47af69862b70ac9

**Pull request closed without merging**